### PR TITLE
Adjusted the info in the security page

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

## Which problem is this PR solving?
- Currently listed server seems to be having problems
- Inconsistencies in the request to encrypt messages

## Short description of the changes
- Use the server pool for the keyserver, even if the TLS certs are typically not matching the request
- Clarified that all communication with us should preferrably be encrypted

### CODE REVIEW

File: content/report-security-issue.md [hunk 1]
```diff
@@ -6,17 +6,18 @@ If you find something suspicious and want to report it, we'd really appreciate!
 
 ## Ways to report
 
+* Prefer to always encrypt your message, no matter the channel you choose to report the issue
```
Comment (objectiser) [inline content/report-security-issue.md:9]: "Always encrypt ...." (same comment as on other PR).

File: content/report-security-issue.md [hunk 2]
```diff
@@ -6,17 +6,18 @@ If you find something suspicious and want to report it, we'd really appreciate!
...
 * Contact us on our open chat room on [Gitter][gitter-room]
-* Send a message to [jaeger-tracing@googlegroups.com][mailing-list]. Ideally, encrypt it using our [published key][published-key], which should match the one shown below.
-* Merge request on GitHub: if you can, fork the affected repository and send us a pull request. We really prefer if you'd talk to us before, though, as our repositories are public and we would like to give a heads up to our users before disclosing it publicly.
+* Send a message to [jaeger-tracing@googlegroups.com][mailing-list]
+* Merge request on GitHub: if you can, fork the affected repository and send us a pull request. We really prefer if you'd talk to us before, though, as our repositories are public and we would like to give a heads up to our users before disclosing it publicly
```
Comment (objectiser) [inline content/report-security-issue.md:12]: Depending on discussion on other PR, this line should be removed.

File: content/report-security-issue.md [hunk 3]
```diff
@@ -6,17 +6,18 @@ If you find something suspicious and want to report it, we'd really appreciate!
...
 
 ## Our PGP key
 
-No matter what channel you choose to communicate with us, feel free to encrypt your message using our [published key][published-key], which should match the one shown below. If you are new to PGP, you can run the following command to encrypt a file called "message.txt":
+No matter what channel you choose to communicate with us, feel free to encrypt your message using our [published key][published-key], available in all major key servers and which should match the one shown below. If you are new to PGP, you can run the following command to encrypt a file called "message.txt":
```
Comment (objectiser) [inline content/report-security-issue.md:16]: Remove "feel free to"? so instructing them to instruct instead of giving option?

Comment (jpkrohling) [reply content/report-security-issue.md]: Changed to "prefer to" instead.

File: content/report-security-issue.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ If you find something suspicious and want to report it, we'd really appreciate!
 
 ## Our PGP key
 
-No matter what channel you choose to communicate with us, feel free to encrypt your message using our [published key][published-key], available in all major key servers and which should match the one shown below. If you are new to PGP, you can run the following command to encrypt a file called "message.txt":
+No matter what channel you choose to communicate with us, prefer to encrypt your message using our [published key][published-key], available in all major key servers and which should match the one shown below. If you are new to PGP, you can run the following command to encrypt a file called "message.txt":
```
Comment (objectiser) [inline content/report-security-issue.md:16]: "we would prefer you to encrypt..."

File: content/report-security-issue.md (hunk 1 above)
Comment (objectiser) [reply content/report-security-issue.md:9]: "Prefer to" -> "It is preferable to...."

**Review by objectiser — COMMENTED** [review]
Sorry a couple more rewording

**Comment by jpkrohling** [comment]
PR updated.

**Review by objectiser — APPROVED** [review]

**Pull request merged**